glslc/test/option_mfmt.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2016 The Shaderc Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import expect
+from glslc_test_framework import inside_glslc_testsuite
+from placeholder import FileShader
+
+MINIMAL_SHADER = '#version 140\nvoid main() {}'
+ERROR_SHADER = '#version 140\n#error\nvoid main() {}'
+
+
```

Comment:
If you want a stronger test, i.e. that checks an exact sequence of words, then you could use the assembler feature of glslc. Then you know *exactly* what words should go into the result string.
Also, that way you can also check the line-wrap functionality while still having a tightly controlled sequence of words, since even an empty SPIR-V module compiles to 5 words.

Optional.